session_5/collegedb/db/schema.rb

```diff
@@ -0,0 +1,52 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2021_02_01_064206) do
```

Comment:
Are you not keeping track of submissions made by a student to an assignment?

Reply:
No not yet, I have just implemented the schema... I wanted to make sure that my schema is correct. By the way, to do that I would need an AssignmentsController right?

Reply:
Which of the tables would a submission be stored in?

You don't need to write any controller or views for this session.

Reply:
Yeah, but I need a create action to save the article right?

Reply:
I would use has_one_attached :assignment or something in the Assignment model right? And then I would use Active Storage to store the assignment right?

Reply:
Ohh, are you suggesting that I implement a method in the Assignment model itself to save the assignment to the database?

Reply:
No, what I am suggesting is far simpler - assignments and submissions are _different_.

A course has many assignments and an assignment has many submissions.

Reply:
Yeah,I got that....